README.md

```diff
@@ -84,6 +84,74 @@ The playbook executes the following steps for you:
 See the image below for an overview of the playbook and roles
 
 
+### Variables
+
+Please note that the variables listed here are not all variables.
+These are the ones that a typical user might want to modify. Check the `defaults/main.yml` files in each role
+to see more internal variables.  
+
+##### Common
+
+|  Variable                     | Description                                                            | Default value      |
+| ----------------------------- | ---------------------------------------------------------------------- | ------------------ |
+| `buildfarm_templates_dir`     | Where to copy the OpenShift templates in Ansible remote                | /tmp               |
+| `project_name`                | Project name in OpenShift to create Jenkins application                | aerogear-digger    |
+| `jenkins_route_protocol`      | Protocol of Jenkins server connection                                  | https              |
+| `concurrent_android_builds`   | Number of max number of concurrent Android builds that are allowed     | 5                  |
+| `deployments`                 | Image definitions for Android slave containers                         |                    |
+| `master_url`                  | OpenShift master URL. Mandatory when using ansible_connection=local    | localhost:8443     |
+| `oc_user`                     | OpenShift user.                                                        |                    |                  
+| `oc_password`                 | OpenShift user password.                                               |                    |                  
+| `jenkins_user`                | Jenkins admin username                                                 | admin              |
+| `jenkins_pass`                | Jenkins admin password                                                 | password           |
+| `master_memory_limit`         | Maximum amount of memory the Jenkins master container can use          | 3Gi                |
+| `master_volume_capacity`      | Volume space available for data                                        | 40Gi               |
+
+
+##### Proxy related
+
+If connections need to go over a proxy in the containers, fill these variables.
```

Comment:
These values are used for external http requests not. The containers are on a no_proxy list so they should communicate without the use of the proxy